Updated `go.sum` after Shipyard's change

mkolesnik opened the pull request:
Since 4ecf8a4f307a4bca8d525773025a465ef230b7f6 in Shipyard removed
support for go 1.16, the `go.sum` is generated differently and hence
needs to be updated.

Signed-off-by: Mike Kolesnik <mkolesni@redhat.com>

<!--  Thanks for sending a pull request!  Here are some tips for you:

1. If this is your first time, please read our developer guide: https://submariner.io/development/
2. Ensure you have added the appropriate tests for your PR: https://submariner.io/development/code-review/#test-new-functionality
3. Read the code review guide to ease the review process: https://submariner.io/development/code-review/
4. If the PR is unfinished, mark it as a draft: https://submariner.io/development/code-review/#mark-work-in-progress-prs-as-drafts
5. If you are using CI to debug, use your private fork: https://submariner.io/development/code-review/#use-private-forks-for-debugging-prs-by-running-ci
6. Add labels to the PR as appropriate.

This template is based on the K8s/K8s template:

https://github.com/kubernetes/kubernetes/blob/master/.github/PULL_REQUEST_TEMPLATE.md
-->

mkolesnik committed:
Updated `go.sum` after Shipyard's change

Since 4ecf8a4f307a4bca8d525773025a465ef230b7f6 in Shipyard removed
support for go 1.16, the `go.sum` is generated differently and hence
needs to be updated.

Signed-off-by: Mike Kolesnik <mkolesni@redhat.com>

mkolesnik commented:
Not sure why E2E (external-net, globalnet) is failing, but it doesn't seem related to this change

tpantelis commented:
> Not sure why E2E (external-net, globalnet) is failing, but it doesn't seem related to this change

```
[11:48:58.358] [submariner]$ [cluster2] docker exec -i ext-app ip r add 10.0.0.0/16 via 172.117.0.3
[11:48:58.673] [submariner]$ [cluster2] docker exec -i ext-app ip r add 10.0.0.0/16 via 172.117.0.3
RTNETLINK answers: File exists
```

The last line does not occur in the non-globalnet job:

```
[07:05:26.392] [submariner]$ [cluster2] docker exec -i ext-app ip r add 10.2.0.0/16 via 172.117.0.3
[07:05:26.708] [submariner]$ [cluster2] docker exec -i ext-app ip r add 10.1.0.0/16 via 172.117.0.3
[07:05:27.028] [submariner]$ [cluster2] docker exec -i ext-app ip r add 100.2.0.0/16 via 172.117.0.3
[07:05:27.356] [submariner]$ [cluster2] docker exec -i ext-app ip r add 100.1.0.0/16 via 172.117.0.3
```

So what does that message mean and what's changed.... @sridhargaddam @mkimuram any ideas?

mkolesnik commented:
@tpantelis OK it seems that the recent "ENV vars" refactor a bit broke this specific testing that relies on `$globalnet`.

#1942 should fix it back, but that one is failing linting because it needs this fix.
I'll add a commit to this PR to fix the var so we can proceed

mkolesnik committed:
Fix broken external net test

This test is relying on `globalnet` variable which isn't set anymore,
instead use GLOBALNET.

Signed-off-by: Mike Kolesnik <mkolesni@redhat.com>
